src/polytope/models/ProblemValidatorTest.py

```diff
@@ -0,0 +1,31 @@
+from dataclasses import dataclass
+from enum import Enum, auto
+
+
+class ProblemValidatorVerdict(Enum):
+    """! Verdicts of problem validators enumeration class."""
+
+    # Input is valid.
+    Valid = auto()
```

Comment:
ditto. Are we covering testlib?

Reply:
I added some more verdicts.

https://github.com/Studio-Polytope/Polytope/blob/6113a64e22145a9de261e9c560d2a0555ba506a0/src/polytope/models/ProblemValidatorTest.py#L5-L16